lib/index.js

```diff
@@ -57,7 +57,7 @@ fs.exists(configLocalFile).then(function (exists) {
         return queuesDispatcher.set(config).dispatch(true);
     }
 }).then(function () {
-    if (!config.endpoints || config.endpoints.length < 1) {
+    if ((!config.endpoints || config.endpoints.length < 1) || (!config.dynamicEndpoints || config.dynamicEndpoints.length < 1)) {
```

Comment:
```suggestion
    if ((!config.endpoints || config.endpoints.length < 1) && (!config.dynamicEndpoints || config.dynamicEndpoints.length < 1)) {
```
This should be an AND instead of an OR (else it will always be thrown when either is empty)